qiskit/circuit/parameterexpression.py

```diff
@@ -538,4 +538,4 @@ def sympify(self):
 
 # Redefine the type so external imports get an evaluated reference; Sphinx needs this to understand
 # the type hints.
-ParameterValueType = Union[ParameterExpression, float]
+ParameterValueType = Union[ParameterExpression, float]  # TODO: should include ParameterVector?
```

Comment:
`ParameterVector` seems to be used in same places as `ParameterValueType` at least sometimes. Possibly we need another type which includes it, or add it to `ParameterValueType`

Reply:
Since `ParameterVector` is a sequence of parameters I think this should not be added here. Do you have an example where both `ParameterValueType` and `ParameterVector` are accepted? 🙂